src/component/selection/getDraftEditorSelectionWithNodes.js

```diff
@@ -155,8 +155,9 @@ function getPointForNonTextNode(
   // wrapper.
   if (editorRoot === node) {
     node = node.firstChild;
+    var win = node && node.ownerDocument.defaultView;
     invariant(
-      node instanceof Element && node.getAttribute('data-contents') === 'true',
+      win && node instanceof win.Element && node.getAttribute('data-contents') === 'true',
```

Comment:
node.nodeType === 1

Reply:
@sophiebits To appease Flow, I needed to change it to:

```js
    node = node.firstChild;
    invariant(
      node && node.nodeType === 1 && typeof node.getAttribute === 'function' &&
      node.getAttribute('data-contents') === 'true',
      'Invalid DraftEditorContents structure.',
    );
```

Does that seem reasonable to you?

Reply:
Note that in this case, because of the `invariant` expression, I can’t really cast `node` as an `Element`, at least not in any way that makes much sense.